### DISCUSSION TITLE
Reaction constant

### DISCUSSION THREAD

**Discussion by heydarimorteza** [GeoChemFoam/GeoChemFoam#49] [General]
I have some questions regarding the value of reaction constant k_c. In one of your papers about GeoChemFOAM (doi: 10.3389/feart.2022.917931), you have used a value of kc=8.9125 × 10–4 m/s for the reaction between acidic flow and calcite.

1. If I am simulating another problem with the reaction occurring only between an acidic flow and a solid surface made up of calcite, should I use the same number for reaction constant? If not, how can I calculate the suitable reaction constant value?
2. If the solid has a density half of that of calcite, what would be the suitable value for k_c in this case?

Thank you.

**Comment by julienmaes** [comment]
Hi Morteza

1. The reaction constant is taken from this paper: https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2011WR011404. It is the product of three terms kc=k1*aH+k2*aCO2+k0. In our work, we assumed that aCO2~0 and that aH is high enough so that k0 can be negelected, but it's perfectly possible to simulate the full thing we GCFOAM
2. Density should not affect k_c. You can change the density and molar mass of the solid in the thermoPhysicalProperties file
